pymc3/tests/test_distributions_random.py

```diff
@@ -221,6 +221,10 @@ class TestTruncatedNormal(BaseTestCases.BaseTestCase):
     distribution = pm.TruncatedNormal
     params = {'mu': 0., 'tau': 1., 'lower':-0.5, 'upper':0.5}
 
+class TestTruncatedNormalLower(BaseTestCases.BaseTestCase):
```

Comment:
While we're here maybe also add a test with upper bound set.

Reply:
Added `class TestTruncatedNormalLower`